### DISCUSSION TITLE
hide-flyout default to "open palette"

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#5813] [Ideas]
**Edit: discussion moved to issue #5821**

I'm using a lot the hide-flyout addon. It works really well and it is extremely useful. To introduce it to my students, that usually move from basic-Scratch to ScratchAddons, it would be better if, by default, the palette would be open, not closed, so that they are not surprised for not seeing the block palette (they are using a version preinstalled in the lab with all the necessary addons already enabled). I guess this could be good for all users (f course if the palette status -closed/open- can be stored in the browser, otherwise closing the palette every time could be annoying).

**Comment by WorldLanguages** [comment]
Sounds good. Can you create an issue? You can reuse the text you already wrote

**Comment by mxmou** [comment]
Related: #1877

**Reply by WorldLanguages** [reply]
I see, in the past we've considered that a bug. But we could add a simple boolean setting for this, I think.
